libs/eventually/src/utils.ts

```diff
@@ -212,3 +213,14 @@ export const validateMessage = (
   }
   return message.data;
 };
+
+/**
+ * Validates admin user
+ *
+ * @param req Request payload
+ * @returns boolean when is an Admin
+ */
+export const isAdmin = (req: Request): boolean | undefined => {
+  const { user } = req as Request & { user: Actor };
+  return user && user.roles.includes("admin");
+};
```

Comment:
this shouldn't be here... eventually doesn't know about express or any other concrete implementation.
eventually-broker on the other hand can provide its own utils and the concept of Admin is only a concern of this package